.env

```diff
@@ -1,2 +1,2 @@
-REACT_APP_BASE_URL=https://cs5610-express-server.herokuapp.com
+REACT_APP_BASE_URL=http://localhost:4000
```

Comment:
We should only use the local address for local development. Revert the change when pushing the code.

Reply:
Perfect! Made this change.